doc/symfony-cache-configuration.rst

```diff
@@ -129,6 +129,29 @@ other hosts, provide the IPs of the machines allowed to purge, or provide a
 RequestMatcher that checks for an Authorization header or similar. *Only set
 one of ``client_ips`` or ``client_matcher``*.
 
+Symfony's `HttpCache` does not support cache invalidation by tags by default.
+However, this library ships with a `TaggableStore` that provides exactly that.
+If you want to use this, adjust your `AppCache` as follows::
+
```

Comment:
can you add a comment that specifies the filename? i guess `// app/AppCache.php`?

Reply:
There are 2-3 examples before that all talk about the `AppCache`, I think it's redundant information :)

Reply:
ah ok, the section above does the same. i find it still useful, but am ok with consistency here for now. might go over that doc at some point again.